CONTRIBUTING.md

```diff
@@ -42,97 +42,109 @@ Merging Pull Requests (for project contributors with write access)
---init --recursive`.
-
-### CMake
-CMake version 3.12 or higher is required to build these libraries. Download and install CMake from the project's
+#### CMake
+CMake version 3.13 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
-### Vcpkg
-Vcpkg is required to download project dependencies. To get started, first clone vcpkg to a location on your system and
-run the bootstrapping script.
+#### Third Party Dependencies
+- curl
+- libxml2
+
+Vcpkg can be used to install the Azure SDK for CPP dependencies into a specific folder on the system instead of globally installing them.
+Follow [vcpkg install guide](https://github.com/microsoft/vcpkg#getting-started) to get vcpkg and install the following dependencies:
 
 ```sh
-git clone https://github.com/Microsoft/vcpkg.git
-cd vcpkg # Keep note of the location of this directory for the next step
-Windows> .\bootstrap-vcpkg.bat
-Linux/macOS:~/$ ./bootstrap-vcpkg.sh
+./vcpkg install curl libxml2
 ```
 
-On macOS, this command may fail if your version of the C++ toolchain is not new enough to support vcpkg. To resolve
-this, vcpkg recommends to install `gcc@6` from Homebrew (`brew install gcc@6`), then re-run the bootstrapping script.
-
-Next, define the `VCPKG_ROOT` environment variable and add the `vcpkg` command to your path. You will probably want to
-persist these changes, so it's recommended to add/edit them via the Windows "System Properties" control panel, or via
-your `.profile` file on Linux/macOS.
-
-> **Windows**
-> ```bat
-> set VCPKG_ROOT=C:\path\to\vcpkg
-> set PATH=%PATH%;%VCPKG_ROOT%
-> ```
->
-> **Linux/macOS**
-> ```sh
-> export VCPKG_ROOT=/path/to/vcpkg
-> export PATH=$PATH:$VCPKG_ROOT
-> ```
-
-Finally, install the project dependencies with vcpkg.
-
-> **Windows**
-> ```bat
-> set VCPKG_DEFAULT_TRIPLET=x64-windows-static
-> vcpkg install curl[winssl]
-> ```
->
-> **Linux/macOS**
-> ```sh
-> vcpkg install curl[ssl]
-> ```
-
-### Development headers (Linux/macOS)
-On Linux/macOS the development headers for OpenSSL 1.1 must be installed to a location where CMake can find them.
-For Ubuntu 18.04 and up, you can install them directly from the main Ubuntu repository with `apt-get`. For macOS, you
-can install them with Homebrew.
-
-> **Linux (Ubuntu 18.04 and up)**
-> ```sh
-> sudo apt-get install libssl-dev
-> ```
->
-> **macOS**
-> ```sh
-> brew install openssl@1.1
-> ```
+When using vcpkg, make sure to set the `VCPKG_ROOT` environment variable to the vcpkg Git repository folder before using `CMake`.
+
+The Azure SDK for C++ uses [this vcpkg release version](https://github.com/Azure/azure-sdk-for-cpp) for continuos integration (CI) building and testing. Make sure to checkout this version when following the next steps for building and running the Azure SDK for C++. Using a newer vcpkg version might still work, however, it is tested.
```

Comment:
```suggestion
The Azure SDK for C++ uses [this vcpkg release version](https://github.com/Azure/azure-sdk-for-cpp/blob/master/eng/vcpkg-commit.txt) for continuos integration (CI) building and testing. Make sure to checkout this version when following the next steps for building and running the Azure SDK for C++. Using a newer vcpkg version might still work, however, if it is tested.
```

Reply:
will fail the CI leg for verify links?
do I need to first create  a PR for that file?

Reply:
No, the file already exists. The link is valid.